src/generators/ui-schema-gen.ts

```diff
@@ -15,13 +15,16 @@ const createLayout = (layoutType: string): Layout => ({
 /**
  * Derives the type of the jsonSchema element
  */
-const deriveType = (jsonSchema: any): string => {
-    if (jsonSchema.type) {
+const deriveType = (jsonSchema: JsonSchema): string => {
+    if (jsonSchema.type && typeof jsonSchema.type === 'string') {
```

Comment:
this is a typeof check, so we check for the type property being a string and not a string array